templates/tink/External-Dependency-Info.txt

```diff
@@ -0,0 +1,217 @@
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
```

Comment:
Are these intentional?  I don't think we need to modify the Tink template set for this PR.

Reply:
I don't know, I copied this from the other bindings in this repo.
Should I make some changes to it?